.coveragerc

```diff
@@ -200,6 +200,9 @@ omit =
     homeassistant/components/linode.py
     homeassistant/components/*/linode.py
 
+    homeassistant/components/lightwave.py
+    homeassistant/components/*/lightwave.py
```

Comment:
Let's follow the embedded platforms model.  See hue or smartthings for reference.  For example, `components/light/lightwave.py` goes to `components/lightwave/light.py`

Reply:
The file I updated was `components/switch/lightwave.py` why is `components/lightwave/light.py` forming part of this code review?

Reply:
If you rebase your code, you'll see the platform was moved and renamed.  What you think was a change to a file came through as an entirely new file.  Just got lucky there weren't conflicts.

Reply:
How do I add myself to CODEOWNERS?

Reply:
I think this has occurred because I don't know how to use GIT. I will close the PR and try again.